RealmBrowser/Classes/RLMApplicationDelegate.m

```diff
@@ -94,6 +96,27 @@ - (BOOL)application:(NSApplication *)application openFile:(NSString *)filename
     return YES;
 }
 
+- (void)application:(NSApplication *)sender openFiles:(NSArray *)filenames
+{
+    if (filenames.count > kMaxNumberOfFilesAtOnce) {
+        NSString *message = [NSString stringWithFormat:@"Are you sure you wish to open all %lu Realm files?", (unsigned long)filenames.count];
+        
+        NSAlert *alert = [[NSAlert alloc] init];
+        [alert setMessageText:message];
+        [alert setInformativeText:@"Opening too many files at once may result in system instability."];
```

Comment:
I agree with Brian that this may be outright the wrong approach, and that a variation on what happens now could be better. Maybe we could just improve the message in the prompt that occurs when a file can't be opened due to too many open files to clarify that this is the case.